code/modules/power/cell.dm

```diff
@@ -369,3 +369,13 @@
 	desc = "A high capacity, slow charging cell for the B.S.G."
 	maxcharge = 40000
 	chargerate = 2600 // about 30 seconds to charge with a default recharger
+
+/obj/item/stock_parts/cell/emproof/reactive // EMP proof so emp_act does not double dip.
+	name = "reactive armor power cell"
+	desc = "A cell used to power reactive armors."
+	maxcharge = 2000
+
+/obj/item/stock_parts/cell/emproof/reactive/small
+	name = "small reactive armor power cell"
+	desc = "A smaller cell used to power reactive armors."
+	maxcharge = 1200
```

Comment:
I feel like the armours should have a energy_cost var instead of using different cells.

overriding a variable is easier than overriding and creating a new type path.

Reply:
sure, one moment